pybamm/models/full_battery_models/base_battery_model.py

```diff
@@ -658,22 +660,6 @@ def set_voltage_variables(self):
         eta_r_av = eta_r_p_av - eta_r_n_av
         eta_r_av_dim = eta_r_p_av_dim - eta_r_n_av_dim
 
-        # SEI film overpotential
-        eta_sei_n_av = self.variables[
-            "X-averaged negative electrode sei film overpotential"
-        ]
-        eta_sei_p_av = self.variables[
-            "X-averaged positive electrode sei film overpotential"
-        ]
-        eta_sei_n_av_dim = self.variables[
-            "X-averaged negative electrode sei film overpotential [V]"
-        ]
-        eta_sei_p_av_dim = self.variables[
-            "X-averaged positive electrode sei film overpotential [V]"
-        ]
-        eta_sei_av = eta_sei_n_av + eta_sei_p_av
-        eta_sei_av_dim = eta_sei_n_av_dim + eta_sei_p_av_dim
-
```

Comment:
lines deleted accidentally?

Reply:
Sorry, I grabbed an older version of develop :/